README.md

```diff
@@ -44,7 +44,7 @@ convert('npm install squirrelly', 'yarn')
 /**
  * Converts between npm and yarn command
  */
-export default function convert (str: string, to: 'npm' | 'yarn' | 'pnpm'): string
+export default function convert (str: string, to: 'npm' | 'yarn' | 'pnpm' | 'bun'): string
```

Comment:
https://github.com/nebrelbug/npm-to-yarn/blob/e787a06e754c9b74f842befd80ba1e91b6c06ae2/src/index.ts#L9